indico/modules/events/abstracts/templates/management/_notification_tpl_list.html

```diff
@@ -95,5 +148,25 @@
                 });
             }
         });
+
+        $('.email-preview-btn').on('click', function() {
+            var id = $(this).data('id');
+            var $previewBtn = $('.email-preview-btn[data-id='+id+']');
+            if ($previewBtn.data('current-text') === 'show') {
```

Comment:
I'd rather use `.data('visible')` and set it to a boolean. Since undefined is falsy you won't even need to clutter the HTML with a default value then.